content/es/guides/directory-structure/pages.md

```diff
@@ -0,0 +1,359 @@
+---
+title: páginas
+description: El directorio de `pages` (páginas) contiene las vistas y rutas de la aplicación. Nuxt.js se encarga de leer todos los archivos de `.vue` dentro de este directorio, y automáticamente crea la configuración de las rutas por ti.
+position: 10
+category: directory-structure
+csb_link: https://codesandbox.io/embed/github/nuxt-academy/guides-examples/tree/master/04_directory_structure/11_pages?fontsize=14&hidenavigation=1&theme=dark
+questions:
+  - question: ¿En cual directorio puedes poner un componente de página?
```

Comment:
"Cual" is inside a question,  so it should be _cuál_ instead.

Reply:
- [ ] fix